albatross/core/model.h

```diff
@@ -112,8 +126,8 @@ class RegressionModel : public ParameterHandlingMixin {
    * properly sized PredictionDistributions.
    */
   PredictionDistribution predict(
-      const std::vector<Feature> &features) const {
-    assert(has_been_fit_);
+      const std::vector<FeatureType> &features) const {
+    assert(has_been_fit());
```

Comment:
assert is fine for now, but in the long run this should be an acceptable failure mode and handled gracefully.